### ISSUE TITLE
Text - Tip is not accessible

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7654]
When the Text component has the `tip` prop defined the tooltip is not accessible to keyboard users. The text needs to be included in the tab order and the Tip needs an `id` in addition to the text having an `aria-describedby` attribute that points to the id for the tooltip to be read out by a screenreader

**Comment by Sulaymon333** [comment]
Tested with Bill and we found out that one of the examples did not announce the tip text. 
I can have a closer look at the issue when we have bandwidth

<img width="611" height="444" alt="Image" src="https://github.com/user-attachments/assets/4b14699f-8aac-493e-949a-cc314e2f278f" />
